src/pages/workspaces/workspace/analysis/AppLauncher.js

```diff
@@ -250,7 +254,6 @@ const ApplicationLauncher = _.flow(
             [runtimeStatus === 'LeoReconfiguring', () => 'Cloud environment is updating, please wait.'],
             [runtimeStatus === 'Error', () => 'Error with the cloud environment, please try again.'],
             [runtimeStatus === null, () => 'Create a cloud environment to continue.'],
-            [runtimeStatus === undefined && runtime === undefined, () => Nav.goToPath(analysisTabName, { namespace, name })],
```

Comment:
Why is this being removed? It seems to make sense to me, if they somehow get to the app launcher with no runtime, we redirect them back to the base page.

Reply:
Hmm this isn't intentionally being removed - I accidentally based my branch off my other ticket. Same with your other comment.
Let me fix this...